### PR TITLE
Removed Must Support from asserter and recorder (aucore-blockvote-2)

### PR DESCRIPTION
**Pull request by dbojicic**
This PR addresses issues from the AU Core R1 Ballot for Comment aucore-blockvote-2: https://confluence.hl7.org/display/HAFWG/aucore-blockvote-2

This PR fixes the following issues: 
- https://jira.hl7.org/browse/FHIR-45019
- https://jira.hl7.org/browse/FHIR-45127
- https://jira.hl7.org/browse/FHIR-45083
- https://jira.hl7.org/browse/FHIR-45082
- https://jira.hl7.org/browse/FHIR-44699
- https://jira.hl7.org/browse/FHIR-45082
- https://jira.hl7.org/browse/FHIR-45066

Changes: 
- removed Must Support from Condition.asserter in AU Core Condition
- removed Must Support from Condition.recorder in AU Core Condition
- removed Must Support from AllergyIntolerance.asserter in AU Core AllergyIntolerance
- removed Must Support from AllergyIntolerance.recorder in AU Core AllergyIntolerance
- removed recorder and asserter from the three AllergyIntolerance examples
- updated Change Log

### CODE REVIEW

File: input/resources/au-core-condition.xml [hunk 1]
```diff
@@ -130,17 +130,6 @@
       </type>
       <mustSupport value="true"/>
     </element>
-    <element id="Condition.asserter">
```
Comment (dt-r) [inline input/resources/au-core-condition.xml:133]: Missing removal of Condition.remover

**Review by dt-r — CHANGES_REQUESTED** [review]
In addition to the comments, need to change general-requirements.md to remove references to asserter and recorder.

File: input/resources/au-core-condition.xml (hunk 1 above)
Comment (dbojicic) [reply input/resources/au-core-condition.xml:133]: Done

**Comment by dbojicic** [comment]
> In addition to the comments, need to change general-requirements.md to remove references to asserter and recorder.

Done

**Review by dt-r — APPROVED** [review]

**Pull request merged**